File: Car-To-Go/Car-To-Go/Models/Purchase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Car_To_Go.Models
{
    public class Purchase
    {
        public Purchase(Model model, Brand brand)
        {
            Model = model;
            Brand = brand;
        }

        public Purchase(Car car, Model model, Photo photo, Status status, Seller seller, Condition condition)
        {
            Car = car;
            Model = model;
            Photo = photo;
            Status = status;
            Seller = seller;
            Condition = condition;
        }

        public Model Model { get; set; }
        public Brand Brand { get; set; }
        public Photo Photo { get; set; }
        public Status Status { get; set; }
        public Seller Seller { get; set; }
        public Condition Condition { get; set; }
        public Car Car { get; set; }

    }
}

File: app-cartogo/src/app/modelos/purchase.ts
import { Model } from './model';
import { Brand } from './brand';
import { Photo } from './photo';
import { Status } from './status';
import { Seller } from './seller';
import { Condition } from './condition';
import { Car } from './car';

export class Purchase{
     Model :Model ;
     Brand :Brand ;
     Photo :Photo ;
     Status :Status ;
     Seller :Seller ;
     Condition: Condition ;
     Car: Car ;
}